packages/appsync-modelgen-plugin/src/visitors/appsync-visitor.ts

```diff
@@ -293,6 +295,7 @@ export class AppSyncModelVisitor<
     };
   }
   processDirectives() {
+    this.processV2KeyDirectives();
```

Comment:
Can we move this inside the `if (this.config.usePipelinedTransformer || this.config.transformerVersion === 2)` conditional below?

Reply:
sure. It's a noop if the new directives aren't present but I can move it inside the if statement